Resources/Prototypes/Entities/Objects/Specific/Mech/mechs.yml

```
@@ -239,7 +241,7 @@
         Blunt: 14 #intentionally shit so people realize that going into combat with the ripley is usually a bad idea.
   - type: MovementSpeedModifier
     baseWalkSpeed: 2.5
-    baseSprintSpeed: 3.8
+    baseSprintSpeed: 5.5 #sunrise-edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)